src/scripts/main.js

```diff
@@ -18,10 +17,23 @@ const food = {
   },
 };
 
-const tree = document.querySelector('#tree');
+const tree = document.querySelector('body');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+    ul.appendChild(li);
+
+    if (typeof data[key] === 'object' && Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
+  }
+
+  element.appendChild(ul);
 }
```

Comment:
The `createTree` function creates an `ul` element and appends `li` elements to it for each key in the data object. If the value of the key is an object and it has keys, the function calls itself recursively. However, it does not append the resulting `ul` from the recursive call to the current `li`. This means that nested objects will not be represented correctly in the resulting HTML tree. To fix this, you should append the result of the recursive call to the current `li`. You can do this by modifying the `createTree` call in line 32 to `li.appendChild(createTree(li, data[key]));`. Additionally, `createTree` should return the created `ul` at the end of the function.